sentry-types/src/protocol/envelope.rs

```diff
@@ -153,11 +155,21 @@ impl Envelope {
                 EnvelopeItem::Transaction(transaction) => {
                     serde_json::to_writer(&mut item_buf, transaction)?
                 }
+                EnvelopeItem::Attachment(attachment) => {
+                    attachment.to_writer(&mut item_buf)?;
```

Comment:
maybe write this to `writer` directly instead of basically copying it into `item_buf`? I think down the road we want to stream this stuff out in the transport.